django-stubs/template/backends/jinja2.pyi

```diff
@@ -17,4 +18,11 @@ class Origin:
     template_name: str | None
     def __init__(self, name: str, template_name: str | None) -> None: ...
 
+class Template:
+    template: Incomplete
+    backend: Jinja2
+    origin: Origin
+    def __init__(self, template: Incomplete, backend: Jinja2) -> None: ...
+    def render(self, context: Incomplete | None = ..., request: Incomplete | None = ...) -> Incomplete: ...
```

Comment:
Is there a reason why these Jinja changes are in this PR? Seems like a separate topic

Reply:
Yes, new version of `pytest-mypy-plugins` requires `jinja2`. Now the environment has changed, new errors are detected by `stubtest`. See https://github.com/typeddjango/django-stubs/issues/1602

Reply:
Oh I see. But we could merge splitted Jinja2 changes first and then this one separately.

Reply:
Or if you don't think it's worth the effort, at least mention both in the PR title and commit message lead line